react/solution/client-details/package.json

```diff
@@ -17,7 +17,7 @@
         "web-vitals": "^1.0.1"
     },
     "scripts": {
-        "start": "react-scripts start",
+        "start": "cross-env PORT=3003 react-scripts start",
```

Comment:
Is there any advantage in using this additional dependency just for specifying different ports for the different apps? We can:
- use a direct command: "set PORT=3003 && react-scripts starts"
- or use a .env file with an environment variable holding the port